pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -177,6 +183,8 @@
   call_to_action_see_module_details: "See module details"
   call_to_action_start_modules: "Start modules"
   call_to_action_see_course_details: "See course details"
+  call_to_action_learn_about_approach: "Learn more about our approach"
+  call_to_action_try_java_lab: " Try Java Lab"
```

Comment:
Is that space at the beginning of " Try Java Lab" intended?

Reply:
Nope, just fixed! Do you have a method/tool that helps find whitespace errors?